duckduckgo/iOS
# tracker radar itests

Task/Issue URL: 
* https://app.asana.com/0/414235014887631/1175383529814450
* https://app.asana.com/0/1163321984198618/1175573289211366

Tech Design URL:
CC:

**Description**:

* Updates the project to remove old test schemes. 
* Renames AtbIntegration tests to IntegrationTests.
* Adds an integration test for Tracker Radar.
* Updates embedded tracker radar and endpoint URL
* Adds a nightly job to Jenkins at https://jenkins.duckduckgo.com/view/iOS/job/ios%20nightly%20tests/

Note that Bitrise will require a workflow change once this is merged to rename the integration tests.  This is being tested with manual builds against a slightly modified workflow.  See https://app.bitrise.io/build/54aaaa418d117565#?tab=log

Note that the Jenkins build will require a tweak to point at `develop` instead of the dev branch.

**Steps to test this PR**:
1. Select Integration Tests and run them, they should pass. 
1. Check that ATB integration tests and Tracker Radar integration tests all ran and passed

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
